spec/label_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../label'
+require_relative '../book'
+
+describe Label do
+  before(:each) do
+    @label = Label.new('Gift', 'blue')
+  end
+  it 'Verify the title of Label' do
+    title = 'Gift'
+    expect(@label.title).to eq(title)
+  end
+
+  it 'Verify the color if the Label' do
+    color = 'blue'
```

Comment:
You don't need to store strings in variables, you can use the strings directly inside the `.to eq()` :mag: